ext/net/01_net.js

```diff
@@ -385,6 +387,11 @@ class Datagram {
   async send(p, opts) {
     switch (this.addr.transport) {
       case "udp":
+        if (p.length > UDP_DGRAM_MAXSIZE) {
```

Comment:
we could just take this check out and leave the PR with the change above. if we try to send with a payload greater than whatever the OS allows (it seems macs define a much lower max size by default we will throw back a system error. i can modify the test to handle this correctly.